src/Aspire.ProjectTemplates.net8/templates/aspire-mstest/.template.config/localize/templatestrings.ja.json

```diff
@@ -1,7 +1,7 @@
 ﻿{
   "author": "Microsoft",
-  "name": ".NET Aspire テスト プロジェクト (NUnit)",
-  "description": ".NET Aspire AppHost プロジェクトの NUnit 統合テストを含むプロジェクト。",
+  "name": ".NET Aspire 9 テスト プロジェクト (MSTest)",
```

Comment:
This is the NUnit template right?

Reply:
Oh it might be something funky with how git/GitHub is detecting this change. It's showing it as a rename of a file in the NUnit template to an MSTest file. Is that intended?

Reply:
```diff
diff --git a/src/Aspire.ProjectTemplates.9.0.net8/templates/aspire-nunit/.template.config/localize/templatestrings.ja.json b/src/Aspire.ProjectTemplates.net8/templates/aspire-mstest/.template.config/localize/tem>
similarity index 75%
rename from src/Aspire.ProjectTemplates.9.0.net8/templates/aspire-nunit/.template.config/localize/templatestrings.ja.json
rename to src/Aspire.ProjectTemplates.net8/templates/aspire-mstest/.template.config/localize/templatestrings.ja.json
index 9dd08c651d..9a12f27d50 100644
--- a/src/Aspire.ProjectTemplates.9.0.net8/templates/aspire-nunit/.template.config/localize/templatestrings.ja.json
+++ b/src/Aspire.ProjectTemplates.net8/templates/aspire-mstest/.template.config/localize/templatestrings.ja.json
@@ -1,7 +1,7 @@
 {
   "author": "Microsoft",
-  "name": ".NET Aspire テスト プロジェクト (NUnit)",
-  "description": ".NET Aspire AppHost プロジェクトの NUnit 統合テストを含むプロジェクト。",
+  "name": ".NET Aspire 9 テスト プロジェクト (MSTest)",
+  "description": ".NET Aspire 9 AppHost プロジェクトの MSTest 統合テストを含むプロジェクト。",
   "symbols/Framework/description": "プロジェクトのターゲット フレームワークです。",
   "symbols/Framework/choices/net8.0/description": "ターゲット net8.0",
   "symbols/skipRestore/description": "指定した場合、作成時にプロジェクトの自動復元がスキップされます。",
```

I think this is git not being able to match the renames correctly. I will check that all the descriptions match the filenames.

Reply:
Yeah, looks like it matches `nunit`->`mstest` for that file, and the opposite also;
```diff
diff --git a/src/Aspire.ProjectTemplates.9.0.net8/templates/aspire-mstest/.template.config/localize/templatestrings.ja.json b/src/Aspire.ProjectTemplates.net8/templates/aspire-nunit/.template.config/localize/tem>
similarity index 75%
rename from src/Aspire.ProjectTemplates.9.0.net8/templates/aspire-mstest/.template.config/localize/templatestrings.ja.json
rename to src/Aspire.ProjectTemplates.net8/templates/aspire-nunit/.template.config/localize/templatestrings.ja.json
index 68f386e5a3..56fa7f8d9b 100644
--- a/src/Aspire.ProjectTemplates.9.0.net8/templates/aspire-mstest/.template.config/localize/templatestrings.ja.json
+++ b/src/Aspire.ProjectTemplates.net8/templates/aspire-nunit/.template.config/localize/templatestrings.ja.json
@@ -1,7 +1,7 @@
 {
   "author": "Microsoft",
-  "name": ".NET Aspire テスト プロジェクト (MSTest)",
-  "description": ".NET Aspire AppHost プロジェクトの MSTest 統合テストを含むプロジェクト。",
+  "name": ".NET Aspire 9 テスト プロジェクト (NUnit)",
+  "description": ".NET Aspire 9 AppHost プロジェクトの NUnit 統合テストを含むプロジェクト。",
   "symbols/Framework/description": "プロジェクトのターゲット フレームワークです。",
   "symbols/Framework/choices/net8.0/description": "ターゲット net8.0",
   "symbols/skipRestore/description": "指定した場合、作成時にプロジェクトの自動復元がスキップされます。",
```